pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dto/EventConfigurationDto.java

```diff
@@ -13,332 +16,106 @@
 import org.broadinstitute.ddp.model.event.NotificationType;
 import org.broadinstitute.ddp.model.event.PdfAttachment;
 import org.broadinstitute.ddp.model.user.EnrollmentStatusType;
+import org.jdbi.v3.core.mapper.reflect.ColumnName;
+import org.jdbi.v3.core.mapper.reflect.JdbiConstructor;
 
 /**
  * A DTO representing the event configuration, left joined with possible trigger and action configuration info
  */
+@Value
+@AllArgsConstructor(onConstructor = @__(@JdbiConstructor))
 public class EventConfigurationDto {
-    private long eventConfigurationId;
-    private EventTriggerType eventTriggerType;
-    private EventActionType eventActionType;
-    private Integer postDelaySeconds;
-    private boolean dispatchToHousekeeping;
-    private String preconditionExpression;
-    private String cancelExpression;
-    private Integer maxOccurrencesPerUser;
-    private int executionOrder;
-    private String label;
-    private String gcpTopic; // FIXME should not be a base field of EventAction
+    long eventConfigurationId;
+    String label;
+    EventTriggerType eventTriggerType;
+    EventActionType eventActionType;
+    Integer postDelaySeconds;
+
+    @Accessors(fluent = true)
```

Comment:
[@Accessors](https://projectlombok.org/features/experimental/Accessors) is considered an experimental feature. It looks like the feature is on-track to being moved to `stable`, but there are potentially backwards incompatible changes [in-flight](https://github.com/projectlombok/lombok/issues/3105). Is `@Accessors` a must here, or can similar effects be achieved using the current [Stable](https://projectlombok.org/features/all) features?

Reply:
@bskinner I think that the alternative is to have the accessors for instance variables of type boolean to use the `is` prefix (the Java bean standard), in this case `isDispatchToHousekeeping()` which I believe you appropriately flagged as reading awkward.
Again, I am inclined to go forward with its usage as I think worst case we can globally/mechanically refactor code to be whatever the new thing is.

Reply:
I'm good with that- My confusion here was misunderstanding the role of the `@Accessors` annotation. It's just controlling how the `@Getter` and `@Setter` annotations synthesize the methods.

My understanding is that the behavior preventing this from being declared a `Stable` feature is a change in how the configuration are inherited from parent declarations and AFAIK we aren't using that. My question was whether `@Accessors` was the only option. It looks like the alternative is setting the global `lombok.accessors.fluent` configuration to `true`, but that also disables the prefixing of setters with `set` so it's not a usable option in this case.

Reply:
Definition of the global variable requires us to change the field names accordingly. Otherwise, getters and setters will be generated a bit differently. From my POV using @Accessors is the most elegant way to solve this. As for the alternatives, there are several:
1. Define methods on our own. 
Let say we have Boolean writeOnce. Lombok will generate getWriteOnce & setWriteOnce(). In addition to that we could manually define shouldWriteOnce() method.
2. Rename all the fields and methods. In this case we will not have shouldXXX canXXX we have to give the readable name starting with is